release_images.yml

```diff
@@ -2,78 +2,32 @@
 release_images:
 # Check .release_images_template.yml for reference
   1:
-    framework: "huggingface_pytorch"
-    version: "1.10.2"
+    framework: "tensorflow"
+    version: "2.7.1"
```

Comment:
Please create a separate entry for this, rather than replacing the HF PT 1.10 entry